### PR TITLE
OCPQE-27370: Enable mini-perm tests on 4.18 AWS jobs

### PR DESCRIPTION
**Pull request by yunjiang29**
Enable mini-perm tests on following jobs:

```
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-disc-priv-sts-efs-mini-perm-f14
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-disc-priv-sts-efs-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-efa-pg-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-efa-pg-mini-perm-f5
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-localzone-byo-subnet-ovn-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-localzone-byo-subnet-ovn-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-localzone-fips-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-localzone-fips-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-private-shared-vpc-phz-sts-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-private-shared-vpc-phz-sts-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-shared-vpc-phz-sts-fips-openldap-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-shared-vpc-phz-sts-fips-openldap-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-wavelength-byo-subnet-ovn-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-wavelength-byo-subnet-ovn-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-wavelength-fips-mini-perm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-wavelength-fips-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-workers-marketplace-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-aws-ipi-workers-marketplace-mini-perm-f5
periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-disc-priv-sts-basecap-none-mini-perm-f14
periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-disc-priv-sts-ep-mini-perm-f14
periodic-ci-openshift-openshift-tests-private-release-4.18-arm64-nightly-aws-ipi-disc-priv-sts-ep-mini-perm-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-iam-profile-fips-mini-perm-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-iam-profile-mini-perm-arm-f7
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-iam-role-efs-arm-regen-cert-f14
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-iam-role-efs-fips-mini-perm-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-iam-role-efs-mini-perm-arm-f2-custom-cert
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-kms-etcd-encryption-fips-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-kms-etcd-encryption-mini-perm-arm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-byo-route53-compact-cloudfront-mini-perm-arm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-ingress-lbtype-classic-subnets-mini-perm-arm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-ingress-lbtype-nlb-subnets-mini-perm-arm-amd-day0-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-ovn-ipsec-mini-perm-amd-arm-day0-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-ovn-ipsec-mini-perm-arm-mixarch-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-proxy-sts-mini-perm-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-proxy-sts-mini-perm-arm-f2
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-public-ipv4-pool-mini-perm-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-public-ipv4-pool-mini-perm-arm-f7
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-usertags-custom-sg-fips-amd-f28-destructive
periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-usertags-custom-sg-mini-perm-arm-f5
```

### CODE REVIEW

**Comment by yunjiang29** [comment]
@jianlinliu @gpei PTAL

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml [hunk 1]
```diff
@@ -163,7 +165,7 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-efs
-- as: aws-ipi-disc-priv-sts-ep-f14
+- as: aws-ipi-disc-priv-sts-ep-mini-perm-f14
   cron: 58 9 7,21 * *
   steps:
     cluster_profile: aws-qe
```
[the remark this answers is not in the document]
Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:171]: weird, let me update/rebase the code, thanks

[the remark this answers is not in the document]
Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:171]: it's in https://github.com/openshift/release/pull/59264/files#diff-6c169299182ad58ceb55b11fa74ecec2b6af6a527e0fdb8fa40ed822459cb456R175-R176

[the remark this answers is not in the document]
Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:171]: you will see them after unfolding the code, they comes from last PR.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml [hunk 2]
```diff
@@ -408,17 +397,19 @@ tests:
...
       BASE_DOMAIN: qe.devcluster.openshift.com
       EXTRACT_MANIFEST_INCLUDED: "true"
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
-- as: aws-ipi-proxy-sts-arm-f2
+- as: aws-ipi-proxy-sts-mini-perm-arm-f2
   cron: 55 12 1,3,5,7,9,11,13,15,17,19,21,23,25,27,29 * *
   steps:
     cluster_profile: aws-qe
```
[the remark this answers is not in the document]
Comment (yunjiang29) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml:415]: same as https://github.com/openshift/release/pull/59264#discussion_r1859787737

**Comment by jianlinliu** [comment]
@yunjiang29 sounds like we also need to enable them on 4.18 z-stream upgrade jobs to keep consistent.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gpei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml:171]: ack, got it

**Comment by jianlinliu** [comment]
cc @liangxia to review

**Comment by yunjiang29** [comment]
@liangxia PTAL, thanks

**Comment by yunjiang29** [comment]
> @yunjiang29 sounds like we also need to enable them on 4.18 z-stream upgrade jobs to keep consistent.

Sure, I will create another PR to do that

**Comment by gpei** [comment]
@yunjiang29 The `core-valid` job failed for the conflicts on the following job yamls:
```
{"command":"PWD=/go/src/github.com/openshift/release GIT_AUTHOR_DATE=1732900367 GIT_COMMITTER_DATE=1732900367 git merge --no-ff 27a71ef6a82e0b161bdd73c7b0e8000010413f17","component":"clonerefs","error":"exit status 1","file":"sigs.k8s.io/prow/pkg/pod-utils/clone/clone.go:85","func":"sigs.k8s.io/prow/pkg/pod-utils/clone.Run.func2","level":"info","msg":"Ran command","output":"Auto-merging ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18-periodics.yaml\nCONFLICT (content): Merge conflict in ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18-periodics.yaml\nAuto-merging ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml\nAuto-merging ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml\nAutomatic merge failed; fix conflicts and then commit the result.\n","severity":"info","time":"2024-11-29T18:09:19Z"}
```

**Pull request merged**